apps/labs/posts/joining_labs.md

```diff
@@ -0,0 +1,71 @@
+---
+title: 'Thoughts on joining Quansight Labs'
+published: April 8, 2019
+author: ralf-gommers
+description: "Ralf Gommers's thoughts on joining Quansight Labs as its director, and his perspectives for pushing forward the Python ecosystem for scientific computing and data science, and how to fund it."
+category: [Community, PyData Ecosystem]
+featuredImage:
+  src: /posts/joining_labs/community_landscape.png
+  alt: 'Image of a word cloud emphasizing the interrelationship of Projects with Assets, Funding and Community.'
+hero:
+  imageSrc: /posts/joining_labs/blog_hero_var1.svg
+  imageAlt: 'An illustration of a brown hand holding up a microphone, with some graphical elements highlighting the top of the microphone.'
+---
+
+In his [blog post](https://www.quansight.com/post/welcoming-ralf-gommers-as-director-of-quansight-labs) welcoming me, Travis set out his vision for pushing forward the Python ecosystem for scientific computing and data science, and how to fund it. In this post I'll add my own perspectives to that. Given that Quansight Labs' purpose, it seems fitting to start with how I see things as a community member and organizer.
```

Comment:
@bskinn this link is broken because of the Quansight blog migration. Would it be the same link after you add this back? If not, could you please correct this, so it will work once you migrate the linked blog post?

Reply:
This link will not need to change. We're migrating all the posts so that they live at the same URLs on the new site as the old.

If you find a link that worked on the old site but doesn't on the new, let me know ASAP. That means either:

1. It's a post that hasn't been migrated yet, has a known link, and should be prioritized; or
2. It's a post that *has* been migrated, but we made a mistake with the URL.